test/cli/t/add-del.t

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+# a TAP compatible test
+
+# quick and dirty: fail early
+set -eu
+
+# test plan
+echo 1..8
+
+# usage:
+#  foo || tapfail
+#  echo "ok"
+# this will tell TAP "not ok" IFF foo has failed
+tapfail() {
+  echo -n "not "
+}
+
+YAST=/usr/sbin/yast
+
+echo "# assume and check that eth0 exists, as a base for the vlan"
+ip addr show dev eth0 || tapfail
+echo "ok 1 eth0: exists already"
```

Comment:
if I get your code in case of fail it returns `not ok 1 eth0: exists already` which is a bit confusing for me.

Another note I have is that I think we should use different directory for such kind of tests as in "test" are unit tests which is run automatic and is non-destructive.